README.md

```diff
@@ -15,14 +19,20 @@ Detailed documentation can be found in our [GitHub Documenation Page](http://vmw
 
 ## Download
 
-**[Click here to download (Github releases)] (https://github.com/vmware/docker-volume-vsphere/releases)**
+**Click here to download from [Github releases](https://github.com/vmware/docker-volume-vsphere/releases)**
```

Comment:
+**[Click here to download latest Github release](https://github.com/vmware/docker-volume-vsphere/releases)**